src/amo/components/AuthenticateButton/index.js

```diff
@@ -98,7 +98,7 @@ export class AuthenticateButtonBase extends React.Component<InternalProps> {
     // mobile browser. This is the cause of
     // https://github.com/mozilla/addons-frontend/issues/1904
     return (
-      <Button
+      <Link
         href={`#${siteUser ? 'logout' : 'login'}`}
         buttonType={buttonType}
```

Comment:
Some of these props (including `buttonType`) are not valid for the `Link` component.